multi-agent/lib/mas/elements/llms/openai/message_converter.py

```
@@ -29,15 +30,22 @@ class OpenAIMessageConverter:
+    def to_openai(
+        messages: List[ChatMessage],
+        sanitizer: Optional[ToolNameSanitizer] = None,
+    ) -> List[Dict[str, Any]]:
         """Convert a list of domain messages to OpenAI API dicts."""
-        return [OpenAIMessageConverter._to_dict(m) for m in messages]
+        return [OpenAIMessageConverter._to_dict(m, sanitizer) for m in messages]
 
     @staticmethod
-    def from_openai(msg: Any) -> ChatMessage:
+    def from_openai(
+        msg: Any,
```

Ruff (0.16.3)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `msg`

(ANN401)